projboard/tests/test_signup.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+from projboard.forms import NewUserForm
+from projboard.models.user import User
+from conftest import (
+                    VALID_USER,
+                    VALID_EMAIL,
+                    VALID_PASSWORD,
+                    INVALID_EMAIL,
+                    INVALID_PASSWORD_MISMATCH,
+                    MISMATCH_MESSAGE_ERROR,
+                    VALID_NAME,
+                    SIGNUP_PATH
+                    )
+
+
+@pytest.mark.django_db
+def test_valid_form():
+    form_data = {
+        'email': VALID_EMAIL,
+        'name': VALID_NAME,
+        'nickname': VALID_USER,
+        'password': VALID_PASSWORD,
+        'password_confirm': VALID_PASSWORD,
+    }
+    form = NewUserForm(data=form_data)
+    if form.is_valid():
+        assert True
+    else:
+        assert False
+
+
+@pytest.mark.django_db
+def test_invalid_form():
+    form_data = {
+        'email': INVALID_EMAIL,
+        'name': VALID_NAME,
+        'nickname': VALID_USER,
+        'password': VALID_PASSWORD,
+        'password_confirm': VALID_PASSWORD,
+    }
+    form = NewUserForm(data=form_data)
+    if form.is_valid():
+        assert False
+    else:
+        assert True
```

Comment:
Can we write here just `assert not form.is_valid()`?

Reply:
Done